src/components/CampaignPage.js

```diff
@@ -678,9 +685,17 @@ class CampaignPage extends Component {
         globals.forEach(element => {
             if(element._id === 'FIATPAYMENT'){
                 this.setState({fiatPaymentEnabled: element.enabled});
+                if(element.enabled){
+                    if(element.CIRCLE && !element.PAYADMIT){
+                        this.setState({fiatPaymentProvider: 'circle'});
+                    } else if (!element.CIRCLE && element.PAYADMIT){
+                        this.setState({fiatPaymentProvider: 'payadmit'});
+                    }
+                }
             }
+            //console.log(element);
         });
-
+        //console.log(this.state.fiatPaymentProvider)
```

Comment:
perhaps, remove this as you did with other log lines

Reply:
done